README.md

```diff
@@ -41,6 +41,89 @@ evolution of the schema. The schema follows [semver](https://semver.org/) versio
 
 This version will not always match the npm package version, as some changes to the npm package might not have changes to the schema. We will publish a list of schema versions and their associated npm versions and git tags.
 
+## Example
+
+Given the following source code:
+
+```js
+class MyElement extends HTMLElement {
+  static get observedAttributes() {
+    return ['disabled'];
+  }
+
+  set disabled(val) {
+    this.__disabled = val;
+  }
+  get disabled() {
+    return this.__disabled;
+  }
+
+  fire() {
+    this.dispatchEvent(new Event('disabled-changed'));
+  }
+}
+
+customElements.define('my-element', MyElement);
+```
+
+The output manifest would look like:
```

Comment:
"Could" sounds ambiguous to me, we're specifying the schema in this repo so it would have to look exactly like the example shown

Reply:
I don't think that necessarily true. For instance, this could contain source references for all the declarations. `disabled` sounds like a boolean, so the field could get a type. Conceivably someone could add extra descriptions to the manifest that aren't in the JS. Tools will have different features for extracting types, descriptions, etc...

Reply:
> For instance, this could contain source references for all the declarations
I wish I knew what a source reference is 😄 

> disabled sounds like a boolean, so the field could get a type.
Not all code is typed

Reply:
I thought I saw an issue from you about SourceReference documentation, but I don't see it now. Do you know where it is?

All values have a type and a developer could certainly write a type in a hand-written manifest. Point is that "could" really is the accurate descriptor here. There are multiple valid manifests per element.